google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -76,21 +76,24 @@ public OSProcessHandler createProcessHandler(
             AppEngineServerModel runConfiguration;
 
             try {
-              // Getting the clone so the debug flags aren't added to the persisted settings.
+              // Getting the clone so the debug com.google.cloud.tools.intellij.flags aren't added
```

Comment:
comment refactor accident?

Reply:
Lol, I thought this is intentional clarification on which flags are used